<launch>
    <arg name="location" />
    <node name="map_saver_lobby" pkg="map_server" type="map_saver" args="-f $(arg location)/lobby" />
</launch>